translation.rst

```diff
@@ -362,6 +343,41 @@ Translations in Templates
 Most of the time, translation occurs in templates. Symfony provides native
 support for both Twig and PHP templates.
 
+.. _translation-filters:
+
+Using Twig Filters
+~~~~~~~~~~~~~~~~~~
+
+The ``trans`` filter can be used to translate *variable texts* and complex expressions:
+
+.. code-block:: twig
+
+    {{ message|trans }}
+
+    {{ message|trans({'%name%': 'Fabien'}, 'app') }}
+
+.. tip::
+
+    You can set the translation domain for an entire Twig template with a single tag:
+
+    .. code-block:: twig
+
+       {% trans_default_domain 'app' %}
+
+    Note that this only influences the current template, not any "included"
+    template (in order to avoid side effects).
+
+By default, the translated messages are output escaped; apply the ``raw``
```

Comment:
I've split the escaping caution to add the relevant information in each section.